cpanfile

```diff
@@ -1,6 +1,13 @@
 use strict;      # satisfy linter
 use warnings;    # satisfy linter
 
+if ( "$]" >= 5.010 ) {
+    requires 'Pod::Man', '>= 5.00';
+}
+else {
+    requires 'Pod::Man', '<= 4.14';
```

Comment:
I think this is going to leave Pod::Man at `1.37` and will then fail to update >= 2.17
maybe rather than `<=` you want an `= 4.14`?

```
Pod::Man (1.37) is installed, but we need version >= 2.17
```
view https://github.com/Perl/docker-perl-tester/actions/runs/3652230751/jobs/6170374554#step:4:1027